whois-api/src/main/java/net/ripe/db/whois/api/rest/mapper/WhoisObjectServerMapper.java

```diff
@@ -71,6 +76,12 @@ public void mapAbuseContact(final WhoisObject whoisObject, final Parameters para
         }
     }
 
+    public void mapObjectInfoMessages(final WhoisObject whoisObject, final Parameters parameters, final RpslObject rpslObject){
+        final List<String> infoMessagesFormatted = Lists.newArrayList();
+        infoMessageValidators.forEach(infoMessageValidator -> infoMessageValidator.generate(rpslObject, parameters, infoMessagesFormatted));
```

Comment:
It should return list of messages and then you can use addAll , it makes i much more independent to other validators/generators?

Reply:
I was following the same logic that we have elsewhere in whois but ok, agree

Reply:
Done